src/Core/CaseFile.java

```diff
@@ -0,0 +1,79 @@
+package Core;
+
+import java.util.List;
+import java.util.Map;
+
+public class CaseFile {
+    private String title;
+    private String invitation;
+    private String description;
+    private String startingRoom;
+    private List<SuspectData> suspects;
+    private List<RoomData> rooms;
+    private List<ExamQuestion> finalExam;
+    private List<String> tasks;
+    private List<String> watsonHints;
+
+    public List<String> getWatsonHints() {return watsonHints;}
+
+    // Nested classes for JSON data
```

Comment:
No reason to nest these classes here

Move to seperate package for json DTO classes